src/main/java/net/earthmc/emcapi/endpoint/towny/NationsEndpoint.java

```diff
@@ -67,6 +68,7 @@ public JsonElement getJsonElement(Nation nation) {
         nationObject.add("coordinates", EndpointUtils.getCoordinatesObject(nation.getSpawnOrNull()));
         nationObject.add("residents", EndpointUtils.getResidentArray(nation.getResidents()));
         nationObject.add("towns", EndpointUtils.getTownArray(nation.getTowns()));
+        nationObject.add("outlaws", EndpointUtils.getResidentArray(NationMetadataManager.getNationOutlaws(nation)));
```

Comment:
The result of getNationOutlaws should be cached in a variable to prevent reading all the metadata lines twice.